plugin/src/steps/preview/index.ts

```diff
@@ -9,7 +9,6 @@ import store from "~/store"
 import { fetchAndCreateSingleNode } from "~/steps/source-nodes/update-nodes/wp-actions/update"
 import { formatLogMessage } from "~/utils/format-log-message"
 import { touchValidNodes } from "../source-nodes/update-nodes/fetch-node-updates"
-import { getDbIdFromRelayId } from "../source-nodes/update-nodes/wp-actions/update"
```

Comment:
I don't see any other changes in this file. Was this import just not used?

Reply:
Yeah I just noticed it was hanging around :p